### ISSUE TITLE
[data-export] Support for multiple SOSL in tabs

### ISSUE THREAD

**Issue by ehsky** [tprouvot/Salesforce-Inspector-reloaded#132]
**Is your feature request related to a problem? Please describe.**
When troubleshooting i usually have a need for multiple SOSL queries on different objects. Today i save them in the query history, but i only need them for a short period so it would be simpler to have an option to have multiple tabs of SOSL. 

**Describe the solution you'd like**
- In the [data-export] on top have tabs for each SOSL you have creation in the current window session. 
Exiting the window will reset the editor to "new" data export state. 
- Have a button on the same line as the tabs mention above named "New export // +" floating to right of existing tabs. When clicked a new temporary tab for a new SOSL is created. 

**Additional context**
![image](https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/26760784/583290eb-5c64-4c89-bd4d-e029e7481f37)

**Comment by sfscgjohnson** [comment]
Would like to see this for SOQL queries as well.

**Comment by tprouvot** [comment]
Yes that could be a good idea! I have to check the effort for those features

**Comment by sshapiro-articulate** [comment]
Hi @tprouvot,

Here is the new feature request: https://github.com/tprouvot/Salesforce-Inspector-reloaded/issues/950

Thanks!
Scott
